batchflow/models/torch/utils.py

```diff
@@ -118,3 +120,48 @@ def center_crop(inputs, target_shape, dims):
     crops_ = [slice(crop_lefts[i], crop_lefts[i] + target_shape[i]) for i in range(dims)]
     crops = [slice(dim) for dim in no_crop_shape] + crops_
     return inputs[crops]
+
+
+def get_blocks_and_activations(model):
```

Comment:
Too much hardcode here: if this code becomes a part of `TorchModel`, it should utilize and use understanding of classes involved. As of now, this code works only for a selected subclass of models.

As model (class `Network`) consists of pre-defined modules (`EncoderModule`, `DecoderModule`, `DefaultModule`, etc), you can check the instance of submodules and extract necessary names.